conda-recipe/meta.yaml

```diff
@@ -76,9 +76,9 @@ test:
         - daal4py/sklearn
     commands:
         - python -c "import daal4py"
-        - mpirun -n 4 python -m unittest discover -v -s tests -p test*spmd*.py # [not win]
-        - mpiexec -localonly -n 4 python -m unittest discover -v -s tests -p test*spmd*.py # [win]
-        - python -m unittest discover -v -s tests -p test*.py
+        - mpirun -n 4 pytest --pyargs --verbose -s tests/test*spmd*.py # [not win]
```

Comment:
is ```--pyargs``` necessary since you are using a path and not a package name? (same goes for the others in ```run_test.bat``` / ```run_test.sh```)

Reply:
I don't know. But all the other `pytest` calls have it.

Reply:
I'd say cut them out and see what happens. If they fail you can readd them (and ping me)

Reply:
Looks like they weren't necessary.